verify.go

```diff
@@ -145,6 +145,13 @@ func (v *IDTokenVerifier) Verify(ctx context.Context, rawIDToken string) (*IDTok
 		}
 	}
 
+	// If a checkExpiry is specified, make sure token is not expired.
+	if v.config.checkExpiry != nil {
```

Comment:
@squat Verifying token expiry is optional, it is implemented only if checkExpiry is set by VerifyExpiry().